indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
-<br>**Detail:** This field is included in XML sitemaps to signal to search engines when a page was last modified. Search.gov collects this metadata in case there is no `article:modified_time` data included in the page itself.
-<br>**Used in:** Indexing processing, page freshness scoring.<br>
-<br>
-<br>
+Details on what we index for PDFs can be found in our [PDF Metadata documentation here]({{site.baseurl}}/indexing/pdf-metadata.html).
+
+<div
+  class="usa-summary-box"
+  role="region"
+  aria-labelledby="summary-box-key-information"
+>
+  <div class="usa-summary-box__body">
+    <h3 class="usa-summary-box__heading" id="summary-box-key-information">
+      Supported Metadata
+    </h3>
+    <div class="usa-summary-box__text">
+      <ul class="usa-list">
+        <li>
+          <a class="usa-summary-box__link" href="#main-page-content">Main Content</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#title">Title</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#description">Description</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#tags--keywords">Tags</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#created-date">Created Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#changed-date">Changed Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#crawling-metadata">Crawling Metadata</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#thumbnail-image">Thumbnail Image</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#audience">Audience</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#content-type">Content Type</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#searchgov-custom-fields">Search.gov Custom Fields</a> 
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+### Fields by Type
+
+|[Standard Fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name)|[OpenGraph Fields](https://ogp.me/)|[DublinCore Fields](https://www.dublincore.org/)|
+|:----|:----|:----|:----|
+|[`<main>`](#main-page-content)|[`og:title`](#title)|[`dcterms.keywords`](#tags--keywords)|
+|[`<title>`](#title)|[`og:description`](#description)|[`dc.subject`](#tags--keywords)|
+|[`description`](#description)|[`article:section`](#tags--keywords)|[`dcterms.subject`](#tags--keywords)|
+|[`keywords`](#tags--keywords)|[`article:tag`](#tags--keywords)|[`dcterms.audience`](#audience)|
+|[`robots`](#crawling-metadata)|[`article:published_time`](#created-date)|[`dc.type`](#content-type)|
+||[`article:modified_time`](#changed-date)|[`dcterms.type`](#content-type)|
+||[`og:image`](#thumbnail-image)|[`dc.date`](#created-date)|
+|||[`dc.date.created`](#created-date)|
+|||[`dcterms.created`](#created-date)|
+
```

Comment:
There's a gray row at the end of the table? 
![Screen Shot 2022-12-09 at 10 51 48 AM](https://user-images.githubusercontent.com/9538411/206740901-6485cea1-f15e-4bab-9fec-a139cab2ecc3.png)

Reply:
This is weird, I don't see that on my computer. What browser are you using?

Reply:
Chrome Version 108.0.5359.98 (Official Build) (x86_64)

Reply:
Ok strange, mine is the same....I'm removing some white space to see if that resolves, but will need your help verified once the updates are reflected in the preview.

Version 108.0.5359.98 (Official Build) (x86_64)

Reply:
Could it be something in the css of our site?

Reply:
still there, let's pair next week.